app/src/main/java/com/bruno/aybar/composechallenges/common/AbsoluteAlignment.kt

```diff
@@ -0,0 +1,36 @@
+package com.bruno.aybar.composechallenges.common
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.LayoutDirection
+import kotlin.math.roundToInt
+
+
+/**
+ * Represents an absolute positioning of a point inside a 2D box. The position will not be
+ * automatically mirrored in Rtl context.
+ */
+@Immutable
+data class AbsoluteAlignment(
```

Comment:
Compose has a `AbsoluteAlignment` class, but the constructor is internal ([filed a bug](https://issuetracker.google.com/issues/169406772)). 

For now, I'm copy-pasting the implementation instead.